.github/ISSUE_TEMPLATE/use-case-issue-template.md

```diff
@@ -0,0 +1,45 @@
+---
+name: Use case issue template
+about: This template must be used for describing new use cases. It can be deleted
+  for general issues.
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+If you are submitting a new USE CASE, please use the template below. Otherwise, delete the use case template before submitting your contribution.
+
+---
+### Use case name
+Status: [In application] ; [Tested] ; [Under development] ; [Idea]
+
+Identifier:
```

Comment:
Do we need this ? The issue number will be sufficient

Reply:
```suggestion
```